packages/core/src/components/Image/index.tsx

```diff
@@ -221,4 +65,106 @@ export default withStyles(() => ({
   backgroundPosition_bottom: {
     backgroundPosition: 'bottom',
   },
-}))(Image);
+});
+
+export type Props = {
+  /** Alternate text if image cannot be displayed */
+  alt: string;
+  /** Specified when image is not directly a part of the content */
+  background?: boolean;
+  /** Sets top or bottom background position */
+  backgroundPositionY?: string;
```

Comment:
also, to be more consistent - boolean props `alignTop` and `alignBottom` instead of a `backgroundPositionY === 'top'` or `backgroundPositionY === 'bottom'`